packages/gatsby/content/features/plugnplay.md

```diff
@@ -128,6 +128,7 @@ Many common frontend tools now support Plug'n'Play natively!
 | Babel | Starting from `resolve` 1.9 |
 | Create-React-App | Starting from 2.0+ |
 | Docusaurus | Starting from 2.0.0-beta.14 |
+| ESBuild | Starting from 0.15.0 |
```

Comment:
https://github.com/yarnpkg/berry/pull/4726 already updates the docs so we'll merge that one since it was opened first (after https://github.com/evanw/esbuild/pull/2453 gets merged and released)